src/components/DeviceConnect/index.js

```diff
@@ -311,14 +309,7 @@ class DeviceConnect extends PureComponent<Props> {
           <Info hasErrors>
             <IconInfoCircle size={12} />
             <Box shrink selectable>
-              {accountName ? (
-                <Trans i18nKey="deviceConnect:info" parent="div">
-                  {'You must use the device associated to the account '}
-                  <strong>{accountName}</strong>
-                </Trans>
-              ) : (
-                String(errorMessage || '')
-              )}
+              {String(errorMessage || '')}
```

Comment:
so, no more markup inside error message? isn't it a regression?

Reply:
we just let the error flow whatever it is, which allow more error cases to appear. we'll just try to skip errors we don't want but ultimately we have more precise error

Reply:
BTW in future, we'll make this displaying on only one line and with ellipsis `...`. the error also need to be selectable